Tools/scripts/build_binaries.py

```diff
@@ -736,6 +736,18 @@ def generate_manifest(self):
         gen_stable.make_all_stable(self.binaries)
         self.progress("Generate stable releases done")
 
+    def generate_firmware_size_json(self):
+        """Generate firmware size json file to download."""
+        self.progress("Generating firmware_size.json")
+        binaries_json_filepath = os.path.join(self.buildlogs_dirpath(),
+                                              "firmware_sizes.json")
+        githash = self.run_git(["rev-parse", "HEAD"]).rstrip()
+        if binaries_json_filepath is not None:
```

Comment:
This will never be the case.  Did you mean to use `os.path.exists`?

Reply:
yep, those are leftover, os.path.isfile is enough